conans/test/functional/command/search_test.py

```diff
@@ -1194,6 +1194,10 @@ class Test(ConanFile):
         client.run("search lib/1.0@user/testing --revisions")
         self.assertIn("bd761686d5c57b31f4cd85fd0329751f (No time)", client.out)
 
+        # test that the pattern search with --revisions enabled works
+        client.run("search li* --revisions")
+        self.assertIn("bd761686d5c57b31f4cd85fd0329751f (No time)", client.out)
```

Comment:
Is it `lib/1.0@user/testing#bd761686d5c57b31f4cd85fd0329751f (No time)`? In that case assert the whole line because it is confusing what is `bd761686d5c57b31f4cd85fd0329751f`